packages/datetime/src/dateInput.tsx

```diff
@@ -298,9 +299,9 @@ export class DateInput extends AbstractComponent<IDateInputProps, IDateInputStat
         const prevMomentDate = this.state.value;
         const momentDate = fromDateToMoment(date);
 
-        // this change handler was triggered by a change in month, day, or (if enabled) time. for UX
-        // purposes, we want to close the popover only if the user explicitly clicked a day within
-        // the current month.
+        // this change handler was triggered by a change in month, day, or (if
+        // enabled) time. for UX purposes, we want to close the popover only if
+        // the user explicitly clicked a day within the current month.
```

Comment:
Wrapped to 80 chars. @adidahiya is there a `tslint` rule or `prettier` rule that can enforce a different max-line-length for comments? Wrapping them at 120 chars makes them kind of hard to read.